ansible/roles/postgresql/tasks/main.yml

```diff
@@ -107,6 +107,20 @@
   when:
     pg_install == True
 
+- name: Create Formplayer database
+  sudo_user: postgres
+  postgresql_db:
+    name: "{{ formplayer_db_name }}"
+    state: present
+    port: 5432
+    owner: "{{ postgres_users.commcare.username }}"
+    encoding: 'UTF-8'
+    lc_collate: 'en_US.UTF-8'
+    lc_ctype: 'en_US.UTF-8'
+    template: 'template0'
+  when:
+    pg_install == True
```

Comment:
Why does this have to be a separate task? Can you not use the existing `create databases` task?